app/controllers/reports_controller.rb

```diff
@@ -19,9 +20,43 @@ def create
     end
   end
 
+  def update
+    if params[:id]
+      @report = Report.find_by id: params[:id]
+      if @report
+        @report.update deleted: true
+        flash.now[:info] = t ".delete_report_success"
+        respond_to :js
+      else
+        flash[:danger] = t ".delete_report_failed"
+      end
+    else
+      flash.now[:danger] = t ".delete_report_failed"
+    end
+  end
+
+  def filter
+    return unless params[:report][:created].present? || params[:status].present?
+
+    @reports = current_user.reports.active_reports.recent_reports
+               .filter_report(params[:report][:created], params[:status])
```

Comment:
```suggestion
    @reports = current_user.reports
.active_reports
.recent_reports 
               .filter_report(params[:report][:created], params[:status])
```
Viết xuống dòng cho dễ nhìn đi e (nhớ sửa indent cho ddusng), tương tự với những chỗ khác nhé

Reply:
Vâng ạ